resources/nl-core/pattern-NlCoreHealthProfessionalReference.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="pattern-NlCoreHealthProfessionalReference" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/pattern-NlCoreHealthProfessionalReference" />
+  <name value="PatternNlCoreHealthProfessionalReference" />
+  <title value="Pattern for zib HealthProfessional References" />

```

Comment:
Should the name be Pattern for **nl-core** HealthProfessional References?

Reply:
I'd rather stick with zib, as this is about references to the zib, not the profile that implements it.

Reply:
Ok!